src/presentation/pages/Note.vue

```diff
@@ -1,43 +1,39 @@
 <template>
-  <div v-if="note === null">
-    Loading...
-  </div>
-  <div v-else>
-    <div>
-      <Button
-        v-if="props.id != null"
-        @click="createChildNote"
-      >
-        {{ t('note.createChildNote') }}
-      </Button>
-    </div>
-    <Editor
-      v-if="isToolsLoaded"
-      ref="editor"
-      :tools="loadedTools"
-      :content="note.content"
-      :read-only="!canEdit"
-      @change="noteChanged"
+  <div>
+    <NoteHeader
+      :id="props.id"
+      :opacity="props.id ? 1 : 0"
     />
```

Comment:
i think we need to move it into v-else
should we really see noteHeader, when note is loading?